src/Dofs/DofHandler.jl

```diff
@@ -126,23 +126,23 @@ function close!(dh::DofHandler{dim}) where {dim}
     # loop over all the cells, and distribute dofs for all the fields
     for (ci, cell) in enumerate(getcells(dh.grid))
         @debug println("cell #$ci")
-        for fi in 1:nfields(dh)
-            interpolation_info = interpolation_infos[fi]
-            @debug println("  field: $(dh.field_names[fi])")
+        for (i,fi) in enumerate(dh.fields)
```

Comment:
Hmmm, I think this is why I didn't use a `Field` struct before, this loop will now be type unstable because all fields in `dh.fields` are not of the same type...

Reply:
Maybe it´s ok since `close` is only called in the initialization of the problem? An other possibility is to add `dim` and `name` of the field to `InterpolationInfo` so we don´t need to loop over `dh.fields` at all.